translations.yml

```diff
@@ -493,6 +521,7 @@
   pt-BR: 'Compartilhe suas ideias, sugestões ou comentários por e-mail para a lista pública [$1](mailto:$1?subject=%s) ou via GitHub.'
   fa: 'لطفا ایده‌ها، پیشنهادات یا نظرات خود را از طریق ایمیل به فهرست بایگانی شده عمومی [$1](mailto:$1?subject=%s)  یا از طریق گیت‌هاب به اشتراک بگذارید.'
   id: 'Mohon berikan gagasan, saran, atau komentar melalui email ke arsip-publik [$1](mailto:$1?subject=%s) atau melalui GitHub.'
+  pl: 'Prosimy o dzielenie się pomysłami, sugestiami lub komentarzami za pośrednictwem poczty elektronicznej na publicznie zarchiwizowanej liście [$1](mailto:$1?subject=%s) lub za pośrednictwem GitHub'
```

Comment:
GitHub or Github? Choosing one and sticking to it would improve consistency.

Reply:
GitHub is best.